data.json

```diff
@@ -449,6 +449,15 @@
     "dateAdded": "2019-10-10T13:00:00.000Z",
     "tags": ["hacktoberfest", "clothing", "expired"]
   },
+  {
+    "name": "GoFr",
+    "difficulty": "medium",
+    "description": "Contribute to GoFr by submitting a notable pull request or by writing an article. contributions in someway to development or spreading the word about GoFr. receive GoFr T-Shirt and Stickers as a token of appreciation.",
```

Comment:
```suggestion
    "description": "Contribute to GoFr by submitting a notable pull request or by writing an article. Contributions in someway to development or spreading the word about 'GoFr'. Receive GoFr T-Shirt and Stickers as a token of appreciation.",
```

Reply:
why is the build failing?

Reply:
I'll update the description as you suggested and change the image url to (https://avatars.githubusercontent.com/u/146108433?s=200&v=4). hope it works.